package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "simorgh",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

```suggestion
  "version": "1.0.0",
```
I dont think were bumping simorgh yet

---

Yes this should stay as-is.